pikaday.js

```diff
@@ -832,10 +832,18 @@
          */
         setMinDate: function(value)
         {
-            setToStartOfDay(value);
-            this._o.minDate = value;
-            this._o.minYear  = value.getFullYear();
-            this._o.minMonth = value.getMonth();
+            if(value) {
```

Comment:
I would check for `value` being a date here, to make sure `getFullYear` and `getMonth` exist.